services/apis/fence/fenceTasks.js

```diff
@@ -111,7 +111,7 @@ module.exports = {
     return I.sendGetRequest(url).then(res => res.body);
   },
 
-  getGoogleFileFromSignedUrlRes(signedUrlRes) {
```

Comment:
i think this function would be better as a 2-step fence **sequence** (similar to `checkFileEquals()`)
- `fenceQuestions.hasUrl(signedUrlRes, FILE_FROM_URL_ERROR)` (you can add optional error message param to `hasUrl`)
- `return await fenceTasks.getFile(signedUrlRes.body.url)`

Reply:
This isn't getting a signed url from fence though, it's actually taking a signed URL response as input, and following the URL to get the actual file contents from cloud storage

Reply:
but what's the difference? the `if` block checks the same thing `hasUrl` does (i guess you can change the error message to make it more general if the url is not from fence) and the `get` does the same thing as `getFile(signedUrlRes.body.url)`

Reply:
good point, okay, will change it to a sequence

Reply:
Copy of our discussion about this for future ref:
> I'm running into issues with it as a sequence, b/c hasUrl uses assertions, but before (for consistency in the tests) I was just getting the response (even if it was failed) and then doing an assert in the test (which I find clearer)
> if I keep it as a sequence, I can only call it when I expect a signedURL and need to check the responses where I expect to fail with essentially the check that was in the existing task
> tldr; so the main difference b/t just using the sequence and old way is that the old way was not doing asserts in the task b/c it's expected to fail for some cases
> it is returning an error message instead of throwing